### PR TITLE
Prepare backend for webp images and change picture url list to Text type

### PR DESCRIPTION
**Pull request by jirehcwe**
In datastore, Strings > 1500 characters aren't supported, so I had to convert our existing datatypes to Text and make it backward compatible.

### CODE REVIEW

**Comment by jirehcwe** [comment]
Photos were also updated to webp in datastore

File: src/main/java/com/google/sps/servlets/BusinessDataServlet.java [hunk 1]
```diff
@@ -200,4 +202,20 @@ public BusinessData(String business, Long ID)
...
   }
 
+  private List<String> getPhotoUrlList(Entity entity) {
+    String[] picturesUrlsArr;
+
+    Object uncastedPhotoList = entity.getProperty(BUSINESS_PICTURES);
+    System.out.println(uncastedPhotoList);
+    if (uncastedPhotoList instanceof Text) {
+      String castedPhotoListJson = ((Text) uncastedPhotoList).getValue();
+      picturesUrlsArr = gson.fromJson(castedPhotoListJson , String[].class);
```
Comment (dalisc) [inline src/main/java/com/google/sps/servlets/BusinessDataServlet.java]: extra space after `castedPhotoListJson`

File: src/main/java/com/google/sps/servlets/EditBusinessDataServlet.java [hunk 2]
```diff
@@ -247,4 +254,19 @@ private float getFloatParameter(HttpServletRequest request, String formElementNa
     }
   }
 
+  private List<String> getPhotoUrlList(Entity entity) {
+    String[] picturesUrlsArr;
+
+    Object uncastedPhotoList = entity.getProperty(BUSINESS_PICTURES);
+    if (uncastedPhotoList instanceof Text) {
+      String castedPhotoListJson = ((Text) uncastedPhotoList).getValue();
+      picturesUrlsArr = gson.fromJson(castedPhotoListJson , String[].class);
```
Comment (dalisc) [inline src/main/java/com/google/sps/servlets/EditBusinessDataServlet.java]: extra space after `castedPhotoListJson`

**Review by dalisc — APPROVED** [review]
very minor stuff otherwise lgtm!

**Pull request merged**